jump.lua

```diff
@@ -78,7 +80,15 @@ jumpdrive.simulate_jump = function(pos, player, show_marker)
 	local is_empty, empty_msg = jumpdrive.is_area_empty(target_pos1, target_pos2)
 
 	if not is_empty then
-		msg = "Jump-target is obstructed (" .. empty_msg .. ")"
+		if core.settings:get_bool("allow_jumpdrive_emerge", false) and empty_msg == "uncharted" then
```

Comment:
```suggestion
		if core.settings:get_bool("jumpdrive.allow_emerge", false) and empty_msg == "uncharted" then
```
Just to follow naming style of other jumpdrive settings.